content/blog/2021-06-25-june-21-community-gems.md

```diff
@@ -0,0 +1,121 @@
+---
+title: June '21 Community Gems
+date: 2021-06-25
+description: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: remote storage integration, hyperparameter tuning,
+  best practices for managing experiments and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, hyperparameter tuning,
+  best practices for managing experiments and more.
+picture: 2021-05-25/gems-cover.png
```

Comment:
looks like image doesn't work for me here https://dvc-org-may-community-g-vuepmc.herokuapp.com/blog ?

Reply:
no image here as well - 

<img width="868" alt="Screen Shot 2021-06-26 at 1 01 05 PM" src="https://user-images.githubusercontent.com/3659196/123524371-9b05bd00-d67e-11eb-97ef-936a1979f19a.png">

Reply:
Got that fixed.